components/app.js

```diff
@@ -28,8 +32,34 @@ class App {
             error: this.handleGetGradesError
         });
     }
+    createGrade(name,course,grade){
+        console.log(name,course,grade);
```

Comment:
Please remove this `console.log`, now that you've verified this functionality works.